portal/server/controllers/dashboard/deals/deals-filters-query.js

```diff
@@ -94,15 +88,24 @@ const dashboardDealsFiltersQuery = (createdByYou, filters, user) => {
        *    { $or: [{dealType: 'GEF}, {dealType: 'BSS'}] },
        *    {status: 'Draft'},
        * ]
+       * if created by you filter, follows different logic so if else statement to check
        */
       if (!isKeywordField && !hasAllStatusesFilterValue) {
         const fieldFilter = {};
         // or for field (eg dealType)
         fieldFilter.$or = [];
         filterValue.forEach((value) => {
-          fieldFilter.$or.push({
-            [fieldName]: value,
-          });
+          // if created by you then adding user id as compared to or statement
+          if (value === CONTENT_STRINGS.DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.DEALS.CREATED_BY_YOU) {
+            // delete or filter as not needed for created by you
+            delete fieldFilter.$or;
+            fieldFilter['maker._id'] = user._id;
+          } else {
+            // else for all other filters apart from createdByYou
+            fieldFilter.$or.push({
```

Comment:
I know this has been there before, just wondering should it not be `$and`?
As we would want all the filters selected as `and`?

Reply:
This one has to be or.  Eg.  if select AIN, MIA and MIN filters, then want it to be AIN or MIA or MIN as AIN and MIA and MIN would not work as deals cannot be all 3.  So the way it is done is (GEF or BSS) and (AIN or MIN or MIA) and (createdbyYou)